public/app/features/dashboard-scene/scene/NavToolbarActions.tsx

```diff
@@ -331,14 +334,21 @@ export function ToolbarActions({ dashboard }: Props) {
         tooltip="Enter edit mode"
         key="edit"
         className={styles.buttonWithExtraMargin}
-        variant="primary"
+        variant={config.featureToggles.newDashboardSharingComponent ? 'secondary' : 'primary'}
```

Comment:
Should we be changing the "Edit" button? We do not own this so I just want to make sure the team that does is okay with this.

Reply:
Yes, they already reviewed the designs and it should be secondary, if the new button is present